observability/distr_tracing/distr_tracing_tempo/distr-tracing-tempo-configuring.adoc

```diff
@@ -17,7 +17,13 @@ endif::[]
 
 include::modules/distr-tracing-tempo-config-default.adoc[leveloffset=+2]
 
-include::modules/distr-tracing-tempo-config-storage.adoc[leveloffset=+2]
+[role="_additional-resources"]
+[id="additional-resources_distr-tracing-tempo-config-default"]
+==== Additional resources
+* link:https://github.com/grafana/tempo-operator/blob/main/docs/spec/tempo.grafana.com_tempostacks.yaml[List of TempoStack CR parameters]
+* link:https://github.com/grafana/tempo-operator/blob/main/docs/spec/tempo.grafana.com_tempomonolithics.yaml[List of TempoMonolithic CR parameters]
+* xref:../../../observability/distr_tracing/distr_tracing_tempo/distr-tracing-tempo-installing.adoc#installing-a-tempostack-instance[Installing a TempoStack instance]
+* xref:../../../observability/distr_tracing/distr_tracing_tempo/distr-tracing-tempo-installing.adoc#installing-a-tempomonolithic-instance[Installing a TempoMonolithic instance]
```

Comment:
```suggestion
* xref:../../../observability/distr_tracing/distr_tracing_tempo/distr-tracing-tempo-installing.adoc#installing-a-tempomonolithic-instance[Installing a `TempoMonolithic` instance]
```

Side note - throughout the docs, I'd probably stick with using either "CR" or "instance" consistently to make it easier to follow that these are the same thing.

Reply:
Thank you.